src/redux/requests/selectedRequestSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const selectedRequestSlice = createSlice({
+  name: "selectedRequest",
+  initialState: {
+    data: "",
+  },
+  reducers: {
+    selectRequest: (state, action) => {
+      state.data = action.payload;
+      console.log("hhhhhhhhhhhhhhhhhhhhhhhhh");
```

Comment:
delete this

Reply:
also when rejecting or accepting please add some kind of loader to show the user something is happening